app/helpers/api/v2/cash_outs_helper.rb

```diff
@@ -36,4 +39,99 @@ def order!(collection)
     _collection
   end
 
+  def send_paypal!()
+    # select cash outs that have been approved but not sent yet
+    collection = ::CashOut::Paypal
+      .where('cash_outs.approved_at IS NOT NULL')
+      .where('cash_outs.sent_at IS NULL')
+      .where('cash_outs.batch_id IS NULL')
+
+    threads = []
+
+    # # Paypal only allows making 500 payouts per request
+    # # so we need to iterate through the collection 500 at a time
+    collection.find_in_batches(batch_size: 500) do |batch|
+      threads << Thread.new { send_paypal_batch(batch) }
+    end
+
+    threads.map(&:join)
```

Comment:
Let's keep this simple and not use threads (we rarely have 500+ payments pending):
```
collection.find_in_batches(batch_size: 500) do |batch|
  send_paypal_batch(batch)
end
```